### ISSUE TITLE
use cloud annotaion in electron 

### ISSUE THREAD

**Issue by mas1515** [cloud-annotations/object-detection-react#22]
hello 
please make object detection tutorial for 
[electron framework](https://www.electronjs.org/)

is it possible use model annotation with electron ??

**Comment by bourdakos1** [comment]
I believe it's possible, but I don't have much experience with electron

**Comment by mas1515** [comment]
@bourdakos1  
i install it 
but i dont know how use it 


` "@cloud-annotations/models": "^0.1.1",
        "wcjs-player": "^6.0.0",
        "wcjs-prebuilt": "^3.0.0"`

**Comment by bourdakos1** [comment]
There’s instructions on how to use the sdk here: https://github.com/cloud-annotations/javascript-sdk

**Comment by mas1515** [comment]
Hello @bourdakos1 

i read javascript-sdk
and work like below but i can't load model 
and get error

i make simple electron app and
my step is here
1- i import model_web folder in my project
2- make a js file , test2.js
3- save this data on it ,https://cdn.jsdelivr.net/npm/@cloud-annotations/models
see image in link
 https://pasteboard.co/J9GQkNv.png

and i  call my model with this code (image uploaded below)

https://pasteboard.co/J9GRjb5.png

and i get this error (image uploaded below)

https://pasteboard.co/J9GSM2Q.png


first line is this 
https://pasteboard.co/J9GW3IV.png

second line is this
https://pasteboard.co/J9GWzfR.png

please help,

**Comment by bourdakos1** [comment]
Hmm, my guess is that it’s having issues locating the model folder

**Comment by mas1515** [comment]
@bourdakos1 

i work more but 
get this two error.
1-

https://pasteboard.co/J9HuM5y.png

2-
and another error that say 
models is not define 
but i have refrenced this  js script 
At the Top locally
https://cdn.jsdelivr.net/npm/@cloud-annotations/models

error image >>
https://pasteboard.co/J9HvKg2.png

**Comment by bourdakos1** [comment]
hmm sorry, I'm not sure I can help

**Comment by mas1515** [comment]
@bourdakos1 
ok man

if it is possible
make electron object detection tutorial in future . 
thank you .

**Comment by mas1515** [comment]
hello @bourdakos1 
is it possible use my web_model 

with tensorflowjs ??? 
for object detection .

is there demo or tutorial ?

in this link 
https://github.com/cloud-annotations/javascript-sdk

you used 
 import models from '@cloud-annotations/models'

not tensorflow js

**Comment by bourdakos1** [comment]
https://github.com/cloud-annotations/javascript-sdk uses tensorflowjs

**Comment by mas1515** [comment]
@bourdakos1  
Hello again how run this project ??
https://github.com/cloud-annotations/javascript-sdk 

i go to its directory   C:\Users\mas\Desktop\models\nodejs\javascript-sdk-master
and run   npm install
next 
npm start but  get this error 
npm ERR! missing script: start

how i test it ???

**Comment by bourdakos1** [comment]
It’s a library so you can’t really run it by itself. You can run the tests though 

npm run btest

**Comment by mas1515** [comment]
@bourdakos1 

i test npm run test   in main directory 
and get this error 



https://pasteboard.co/Jbbpjmy.png


https://pasteboard.co/JbbsNTp.png

**Comment by bourdakos1** [comment]
btest not test

**Comment by mas1515** [comment]
@bourdakos1 
how improve speed and accuracy for object detection when i use javascript sdk  ???

**Comment by bourdakos1** [comment]
Not much you can do to improve speed aside from trying to find a lighter model. Accuracy wise, you just need more training data

**Comment by mas1515** [comment]
@bourdakos1 

can you explain how increase speed for detection ????

What do you mean by lighter model?

**Comment by bourdakos1** [comment]
Instead of using ssd mobile for your model, you could find a model that is faster. That discussion is outside the scope of this issue though and there's not much I can do to help you
